helm/prometheus-rules/templates/alerting-rules/service-level.rules.yml

```diff
@@ -15,16 +15,16 @@ spec:
         description: '{{`Service level burn rate is too high for {{ $labels.service }} service.`}}'
         opsrecipe: service-level-burn-rate-too-high/
       expr: |
-        slo_errors_per_request:ratio_rate1h{service!~"efk-.*|.*external-dns.*|kong-.*|.*nginx-.*"}
-        and on (service)
         (
-        slo_errors_per_request:ratio_rate1h > on (service) group_left (cluster_type, cluster_id, class) slo_threshold_high
+          slo_errors_per_request:ratio_rate1h{service!~"efk-.*|.*external-dns.*|kong-.*|.*nginx-.*"} > on (service) group_left slo_threshold_high
         and
-        slo_errors_per_request:ratio_rate5m > on (service) group_left (cluster_type, cluster_id, class) slo_threshold_high
+          slo_errors_per_request:ratio_rate5m{service!~"efk-.*|.*external-dns.*|kong-.*|.*nginx-.*"} > on (service) group_left slo_threshold_high
+        )
         or
-        slo_errors_per_request:ratio_rate6h > on (service) group_left (cluster_type, cluster_id, class) slo_threshold_low
+        (
+          slo_errors_per_request:ratio_rate6h{service!~"efk-.*|.*external-dns.*|kong-.*|.*nginx-.*"} > on (service) group_left slo_threshold_low
         and
-        slo_errors_per_request:ratio_rate30m > on (service) group_left (cluster_type, cluster_id, class) slo_threshold_low
+          slo_errors_per_request:ratio_rate30m{service!~"efk-.*|.*external-dns.*|kong-.*|.*nginx-.*"} > on (service) group_left slo_threshold_low
```

Comment:
I am clearly missing some context because this list of services seems a bit arbitrary to me?

Reply:
I do not know where those services comes from, and they should probably excluded from the `raw_slo_errors`/`raw_slo_requests` recording rules rather than here, but we just moved them around.

Reply:
I think the correct way to exclude managed apps from the SLO stuff should be to remove the label `giantswarm.io/monitoring_basic_sli: "true"`. At least that's what I get from the documentation.

Reply:
I added a note in the Team Atlas board about improving this. Hope that's sufficient.